concepts/algorithms/covering_edges.py

```diff
@@ -0,0 +1,84 @@
+"""Covering Edges
+
+cf. Carpineto, Claudio, and Giovanni Romano.
+Concept data analysis: Theory and applications.
+John Wiley & Sons, 2004.
+"""
+
+
+def covering_edges(concept_list, context):
+    """Yield mapping edge as ``((extent, intent), (lower_extent, lower_intent))``
+    pairs (concept and it's lower neighbor) from ``context`` and ``concept_list``
+    
+    Example:
+        >>> from concepts import make_context, ConceptList
+        >>> from concepts._common import Concept
+
+        >>> context = make_context('''
```

Comment:
Just wondering if this example is from the paper

Reply:
Nope, I took context from `fcbo` example and manually checked the result. Not sure if good idea, but maybe it will be useful to have some secondary untitest based on compression between upper neighbor and `covering_edges`.

Reply:
I think it's fine as a first stab. We an replace it if we come up with something better.